src/main/kotlin/pl/edu/uj/ii/ksi/mordor/services/upload/session/FileUploadSessionService.kt

```diff
@@ -0,0 +1,21 @@
+package pl.edu.uj.ii.ksi.mordor.services.upload.session
+
+import java.awt.print.Pageable
+import org.apache.commons.io.monitor.FileEntry
+import org.springframework.data.domain.Page
+import org.springframework.stereotype.Service
+
+@Service
+class FileUploadSessionService {
+    fun accept(uploadSession: FileUploadSession) {}
+
+    fun reject(uploadSession: FileUploadSession) {}
+
+    fun getAllFiles(uploadSession: FileUploadSession): List<FileEntry> {
```

Comment:
make this a method on FileUploadSession

Reply:
`getAllFiles` should be implemented with use of the `RepositoryService`. Should such a method be part of dao?